src/models/transactions/checkCash.ts

```diff
@@ -0,0 +1,37 @@
+import { ValidationError } from "../../common/errors";
+import { Amount, IssuedCurrencyAmount } from "../common";
+import { BaseTransaction, verifyBaseTransaction } from "./common";
+
+export interface CheckCash extends BaseTransaction {
+    TransactionType: "CheckCash";
+    CheckID: string;
+    Amount?:	Amount;
+    DeliverMin?: Amount;
+}
```

Comment:
Since a transaction must include either an Amount or DeliverMin, but not both, could we encode that into the type? (Currently our typing allows for neither Amount nor DeliverMin as an option)

Maybe something like: 
```
interface CheckCashAmount extends BaseTransaction {
    TransactionType: "CheckCash";
    CheckID: string;
    Amount: IssuedCurrencyAmount;
 }
 
 interface CheckCashDeliverMin extends BaseTransaction {
    TransactionType: "CheckCash";
    CheckID: string;
    DeliveryMin: IssuedCurrencyAmount;
}

export type CheckCash = (CheckCashAmount | CheckCashDeliveryMin)
```

Reply:
Changed the code to this! Thank you!